mediator/med.F90

```diff
@@ -2010,7 +2015,9 @@ subroutine SetRunClock(gcomp, rc)
 
     if (first_time) then
 
+       ! --------------------
```

Comment:
These decorations do nothing to add to the readability of the code in my opinion.  Can I ask you to standardize the format if you are going to insist on adding them?

Reply:
I disagree. I think that it really helps readability. I agree about standardization - but its not clear to me what it should be. For now I've removed the extra decorations at mentioned in your comment.